packages/providers/providers.yaml

```diff
@@ -7602,6 +7602,43 @@ reddit:
         base_url: https://oauth.reddit.com
     docs: https://docs.nango.dev/integrations/all/reddit
 
+redtail-crm-sandbox:
+    display_name: Redtail CRM (Sandbox)
+    categories:
+        - crm
+    auth_mode: TWO_STEP
+    token_url: https://review.crm.redtailtechnology.com/api/public/v1/authentication
+    token_request_method: GET
+    token_headers:
+        authorization: Basic ${base64(${credentials.apiKey}:${credentials.username}:${credentials.password})}
```

Comment:
would it make more sense to split the description of the token headers into properties describing how it is being formed? At this pace we are going to create how own language and compiler just to describe and generate it.
for example:
```
token_headers:
    header: authorization
    schema: basic
    token: ${credentials.apiKey}:${credentials.username}:${credentials.password}
    encoding: base64
...
```